release-notes/opensearch-security.release-notes-2.6.0.0.md

```diff
@@ -0,0 +1,15 @@
+## 2023-02-28 Version 2.6.0.0
+
+Compatible with OpenSearch 2.6.0
+
+### Enhancements
+
+* Add actions cluster:admin/component_template/* to cluster_manage_index_templates ([#2409](https://github.com/opensearch-project/security/pull/2409))
+* Publish snapshots to maven ([#2438](https://github.com/opensearch-project/security/pull/2438))
+* Integrate k-NN functionality with security plugin ([#2274](https://github.com/opensearch-project/security/pull/2274))
+
+### Maintenance
+
```

Comment:
Do you need Xcontent factory update?

Reply:
Since it is not user facing and a minor change I did not include it. I'm also not sure if core is doing the backport in time that will make the change required. If needed we can add it to here.